src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -52,63 +52,65 @@ export function initiateMedicationDispenseBody(
   session: Session
 ) {
   let dispenseBody = [];
-  medicationRequests.map((medicationRequest) => {
-    let dispense = {
-      resourceType: "MedicationDispense",
-      status: "completed", // might need to change this to appropriate status
-      authorizingPrescription: [
-        {
-          reference: "MedicationRequest/" + medicationRequest.id,
-          type: "MedicationRequest",
-        },
-      ],
-      medicationReference: medicationRequest.medicationReference,
-      medicationCodeableConcept: medicationRequest.medicationCodeableConcept,
-      subject: medicationRequest.subject,
-      performer: [
-        {
-          actor: {
-            reference: session?.currentProvider
-              ? `Practitioner/${session.currentProvider.uuid}`
-              : "",
+  medicationRequests
+    .filter((medicationRequest) => medicationRequest.status === "active")
```

Comment:
this is the only change/addition in this file, the stuff below is "prettier" formatting changes.